WordPress/Classes/Models/AbstractPost.h

```diff
@@ -13,6 +13,7 @@
 @property (weak, readonly) AbstractPost *revision;
 @property (nonatomic, strong) NSMutableSet *comments;
 @property (nonatomic, strong) Media *featuredImage;
+@property (nonatomic, strong) NSString *restorableStatus; // Should be used to store the post's status before its moved to the trash.
```

Comment:
This is pretty minor, but I'd suggest documenting with the same style we use for methods.  We also get the added bonus of not exceeding the recommended line width.